src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -177,6 +177,11 @@
     <PropertyGroup>
         <PackageBuildNumberOverride Condition="'$(PackageBuildNumberOverride)' == '' and '$(_DerivedPackageNumber)' != ''">$(_DerivedPackageNumber)</PackageBuildNumberOverride>
     </PropertyGroup>
+    <!-- we don't yet have runtime packages for these RIDs so exclude them for now -->
+    <ItemGroup>
+      <ExcludedRuntimes Include="fedora.23-x64" />
```

Comment:
@ellismg I didn't realize we had these packages on the feed.  Here's a sample error message.

error : Unable to resolve 'runtime.ubuntu.16.04-x64.runtime.native.System.Net.Security (>= 1.0.1-rc4-24126-06)' for '.NETCoreApp,Version=v1.0 (ubuntu.16.04-x64)

We do include the dotnet.myget.org feed when restoring packages.